core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/regionserver/transactional/TrxTransactionState.java.tmpl

```diff
@@ -77,96 +94,29 @@
  * Holds the state of a transaction. This includes a buffer of all writes, a record of all reads / scans, and
  * information about which other transactions we need to check against.
  */
-public class TrxTransactionState  extends TransactionState{
-
-    static boolean sb_sqm_98_1 = false;
-    static boolean sb_sqm_98_4 = false;
-    static boolean sb_sqm_98_9 = false;
-    static java.lang.reflect.Constructor c98_1 = null;
-    static java.lang.reflect.Constructor c98_4 = null;
-    static java.lang.reflect.Constructor c98_9 = null;
+public class TrxTransactionState extends TransactionState {
 
-    static Class keepDeletedCellsClazz = null;
-    static Class scaninfoClazz = null;
-    static Constructor scaninfoConstructor = null;
-    static Object[] scaninfoArgs = null;
+    static java.lang.reflect.Constructor c1_0 = null;
 
     static {
         String version = VersionInfo.getVersion();// the hbase version string, eg. "0.6.3-dev"
         LOG.info("Got info of Class ScanQueryMatcher for HBase version :" + version);
-
```

Comment:
i don't think there will happen this rid.. cause we have a 1.3 branch to support backward version(hbase 0.98)